### ISSUE TITLE
RFE : Change the 3 export commands present in the compilation commands window.

### ISSUE THREAD

**Issue by Gilletarom** [rgleason/weather_routing_pi#128]
Dave,

In the research compilation commands window, there are 3 commands to export at the bottom. But, after version 1.15.2, these commands are unsuitable. They should be adapted.

Gilletarom.

![2023-08-24 16_42_13-Météo-routage - WeatherRoutingConfiguration xml](https://github.com/rgleason/weather_routing_pi/assets/6445543/d6efb835-3be3-4e5e-a320-5c242f51b104)

**Comment by rgleason** [comment]
@Gilletarom Please be more specific, we have no idea what you are suggesting here. 

> But, after version 1.15.2, these commands are unsuitable. They should be adapted.

Also, this is from Aug 2023, is it still a problem? If not please close.

**Comment by Gilletarom** [comment]
I was trying to say that the "Import", "Export route" and "Export all" commands were not understandable after version 1.15.2. I was probably too imprecise.

**Comment by rgleason** [comment]
@Gilletarom 

Can you please explain futher and in more detail?
How are these too imprecise?

Import  -  What? a route?  a gpx? (I don't see this in the screenshot) (Keep in mind that I don't have a working version of WR here, so I can see the dialog or try or test things)

Export All - I assume this exports all configuration routes? Or does it do something else?  Should it read "Export all Routes?"

Export - I assume this exports routes. Should it read "Export Route"   or "Export selected route

Additionally, I think Sebastien is going to be reducing the number of marks to make the exported routes more manageable.

**Comment by rgleason** [comment]
@Gilletarom  @sebastien-rosset  
I believe Sebastien has completed this. Can OP please close?

**Comment by sebastien-rosset** [comment]
> [@Gilletarom](https://github.com/Gilletarom) [@sebastien-rosset](https://github.com/sebastien-rosset) I believe Sebastien has completed this. Can OP please close?

Yes, this should be addressed with https://github.com/rgleason/weather_routing_pi/pull/223, at least for the default US English version. Translators can provide proper labels for every language.
